x-pack/solutions/security/plugins/security_solution/public/common/lib/search_ai_lake/hooks/integrations/use_enhanced_integration_cards.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { EuiFlexItem, EuiSpacer, EuiBadge } from '@elastic/eui';
+import { installationStatuses, type IntegrationCardItem } from '@kbn/fleet-plugin/public';
+import { SECURITY_UI_APP_ID } from '@kbn/security-solution-navigation';
+import { CONFIGURATIONS_PATH } from '../../../../../../common/constants';
+import { IntegrationsFacets } from '../../../../../configurations/constants';
+import { RETURN_APP_ID, RETURN_PATH } from './constants';
+
+const FEATURED_INTEGRATION_SORT_ORDER = [
+  'epr:splunk',
+  'epr:google_secops',
+  'epr:microsoft_sentinel',
+  'epr:sentinel_one',
+  'epr:crowdstrike',
+];
```

Comment:
NIT: We could define `SEARCH_AI_LAKE_ALLOWED_INTEGRATIONS` in order, so we would not need to have a duplicated array of integrations here

Reply:
@semd I was back and forth about doing it that way. Bonnie mentioned that if/when we add additional integrations to ai4soc, that the ones now would be featured and others would like add on after that. You can see a preliminary design [here](https://www.figma.com/design/DYs7j4GQdAhg7aWTLI4R69/AI4DSOC?node-id=1-87536&p=f&m=dev). So I tried to better future proof. I also thought that the whole allowlist would be more easily maintained as alphabetical. 

Happy to change it though if we think it makes more sense.

Reply:
I see, that makes sense. However, the only thing we know for sure right now are the current requirements. The concept of "featured" integrations does not exist yet. If it is introduced in the future, we can add support for it at that moment, maybe there's a better way to implement it when we know the full specification.

I agree that we should be mindful about potential future requirements, but I would not prematurely add support for things that are not in the requirements. Keep things simple, it would not be the first time an idea does not get implemented.

That said, it's just a NIT, all good, resolving.